### ISSUE TITLE
Change wording for notice of changes to be reverted

### ISSUE THREAD

**Issue by arthurattwell** [electricbookworks/electric-book-gui#90]
We need to change the wording on this notice to something less scary for non-technical users.

![image](https://user-images.githubusercontent.com/1053331/28472159-c28d68d8-6e3f-11e7-9e9e-206d2acb310a.png)

"We've found some files in your project that you don't need. No need to worry. These were probably created by an incomplete output-generation process. We'll remove them for you the next time you commit."

**Comment by arthurattwell** [comment]
This particular case may be common:

![image](https://user-images.githubusercontent.com/1053331/28497571-e14691a4-6f8a-11e7-8eff-b4fd0d81d893.png)

What's happened is that Bundler has updated the `Gemfile.lock` file. This is a consequence of someone using Windows. (Me!) On Windows, Jekyll needs a gem called `wdm`, which is listed in the `Gemfile` list of dependencies with a flag that says 'only on Windows!'. Then, when Bundler executes Jekyll on Windows, it includes `wdm` in the `Gemfile.lock`. 

Then when the EBM (or anyone on a unix system) runs Bundler, it removes `wdm` from the `Gemfile.lock`.

So much for lock files.

The bottom line is that we could get this notice a lot. So first we should make it friendly. Then we can work on some kind of logic to avoid it altogether.

**Comment by arthurattwell** [comment]
As a result of this, on the EBM currectly (on demo), when I try to fetch, I can't because of the uncommitted file:

![image](https://user-images.githubusercontent.com/1053331/28497620-f7776aa6-6f8b-11e7-8cf1-bdb0100f3ab2.png)

**Comment by arthurattwell** [comment]
I have improved user messaging pending in #96.
